gym/envs/toy_text/blackjack.py

```diff
@@ -49,51 +49,50 @@ def is_natural(hand):  # Is this hand a natural blackjack?
 
 class BlackjackEnv(gym.Env):
     """
-    Blackjack is a card game where the goal is to obtain cards that sum to as
-    near as possible to 21 without going over.  They're playing against a fixed dealer.
+    Blackjack is a card game where the goal is to beat the dealer by obtaining cards
+    that sum to closer to 21 (without going over 21) than the dealers cards.
+    The dealer is fixed in this environment.
 
     Card Values:
 
-    - Face cards (Jack, Queen, King) have point value 10.
-    - Aces can either count as 11 or 1, and it's called 'usable ace' at 11.
-    - Numerical cards (2-9) have value of their number.
+    - Face cards (Jack, Queen, King) have a point value of 10.
+    - Aces can either count as 11 (called a 'usable ace') or 1.
+    - Numerical cards (2-9) have a value equal to their number.
 
-    This game is placed with an infinite deck (or with replacement).
-    The game starts with dealer having one face up and one face down card, while
-    player having two face up cards.
+    This game is played with an infinite deck (or with replacement).
+    The game starts with the dealer having one face up and one face down card,
+    while the player has two face up cards.
 
-    The player can request additional cards (hit, action=1) until they decide to stop
-    (stick, action=0) or exceed 21 (bust).
+    The player can request additional cards (hit, action=1) until they decide to stop (stick, action=0)
+    or exceed 21 (bust, immediate loss).
     After the player sticks, the dealer reveals their facedown card, and draws
     until their sum is 17 or greater.  If the dealer goes bust the player wins.
-    If neither player nor dealer busts, the outcome (win, lose, draw) is
+    If neither the player nor the dealer busts, the outcome (win, lose, draw) is
     decided by whose sum is closer to 21.
 
-    The agent take a 1-element vector for actions.
-    The action space is `(action)`, where:
-    - `action` is used to decide stick/hit for values (0,1).
+    ## Action Space
+    There are two actions: stick (0), and hit (1).
 
-    The observation of a 3-tuple of: the players current sum,
-    the dealer's one showing card (1-10 where 1 is ace), and whether or not the player holds a usable ace (0 or 1).
+    ## Observation Space
+    The observation consists of a 3-tuple containing: the players current sum,
```

Comment:
player's

Reply:
dealer's below

Reply:
I'm pretty sure this is correct since it's a possessive, whereas `'s` is a contraction for `___ is`. If you still object, I don't mind changing it. I know it appears like that all the time.

Reply:
Possessive usually means: 's
"possessive of a singular noun is formed by adding an apostrophe and s," - https://www.thepunctuationguide.com/apostrophe.html
Do you agree with this?

I imagine it does appear like that throughout the code base.

Reply:
No prob, done